### ISSUE TITLE
Identify potentially important features

### ISSUE THREAD

**Issue by juanshishido** [juanshishido/college-scorecard#2]
Profile the data set&mdash;get summary statistics and counts on data.

Create a script that outputs this information.

**Comment by brynamo** [comment]
@juanshishido, how can I help with this? What stats are we looking for specifically?

**Comment by juanshishido** [comment]
Thanks, @brynamo. I _started_ creating a list of potentially useful/important variables here https://github.com/juanshishido/college-scorecard/blob/master/ideas.md#variables-of-interest. It's not yet finished. If you can, please quickly review

Once the list is complete, I plan to write a script to get the five number summaries (min, 25%, median, 75%, max) and mean and standard deviations for each. Also, noting the number of missing values.

This will give us a good sense of what data we can/should work with.

**Comment by juanshishido** [comment]
Decided _not_ to create a separate issue.

I've gone through 15 of the 81 pages in `CollegeScorecardDataDictionary-09-12-2015.pdf`. This leaves between 16 and 17 pages for each of us.

Just as a reference, the last variable I have covered is `PCTPELL`.

### TODO

**I** will go through pages 16-31.
@brynamo go over 32-47.
@PratibhaRathore go over 48-64.
@binnie869 go over 65-81.

**Comment by binnie869** [comment]
I think all of us have identified the important variables from our parts so we do have a consolidated list of variables.

Hence, closing this issue.
